src/main/java/pages/BasePage.java

```diff
@@ -59,6 +62,18 @@ public void clickOnElement(By locator) {
         locateElement(locator).click();
     }
 
+    public void clickOnFacebookButton() {
+        locateElement(facebookButtonLocator).click();
+    }
+
+    public void clickOnTwitterButton() {
+        locateElement(twitterButtonLocator).click();
+    }
+
+    public void clickOnLinkedinButton() {
+        locateElement(linkedinButtonLocator).click();
+    }
```

Comment:
@marikacode There's a method in `BasePage` class called `clickOnElement`, which does exactly that - locates the element and clicks on it. You can use it here